packages/core/src/components/graphs/bar.ts

```diff
@@ -1,16 +1,21 @@
 // Internal Imports
 import { Component } from "../component";
+import { DOMUtils } from "../../services";
 
 export class Bar extends Component {
 	// Gets the correct width for bars based on options & configurations
 	protected getBarWidth(allDataLabels?: string[]) {
 		const options = this.model.getOptions();
+		const numData = this.model.getData().length;
```

Comment:
```suggestion
		const numberOfData = this.model.getData().length;
```

Reply:
I'd even suggest `numberOfDatapoints`, and shouldn't we be using `this.model.getDisplayData()` here instead?

Reply:
Changes made.